src/screens/TopicPageScreen/index.js

```diff
@@ -337,6 +337,7 @@ const TopicPageScreen = (props) => {
 
   const onRefresh = () => {
     refreshingData(0);
+    getTopicDetail();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getTopicDetail` function is invoked without passing the required `domain` parameter. This could lead to unexpected behavior or errors. Please ensure that the necessary arguments are provided when calling this function.

```diff
- getTopicDetail();
+ getTopicDetail(someDomain); // replace `someDomain` with the actual domain value
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->